pkg/ccl/backupccl/backup_planning.go

```diff
@@ -979,9 +979,10 @@ func backupPlanHook(
 
 		// TODO (lucy): For partitioned backups, also add verification for other
```

Comment:
extremely nit-y and I'm sorry, but arguably this comment should stick with the call to the second half of your now split method?

(and if you do opt to move it, let's bump the name to a current team-member i.e. me or you ?)

Reply:
Done.